unlock-app/src/components/interface/checkout/main/Select.tsx

```diff
@@ -128,7 +128,7 @@ const LockOption = ({ disabled, lock }: LockOptionProps) => {
                   )}
                   {formattedData?.formattedKeysAvailable !== 'Unlimited' && (
                     <LabeledItem
-                      label="Quantity"
```

Comment:
The checkout is used by many more things that are mot tickets so we can't set "Tickets" but can we just use "Available" ?

Reply:
This is a good example where having a separate repo would make sense to avoid such issue.
It's always annoying where you know what the right thing is for the end user, but you just cannot do it.
Anyway.

2nd best option is to show "10 left" like on the right of this screenshot. But then we cannot reuse the same <LabeledItem> component.
![photo_2023-08-14 14 29 45](https://github.com/unlock-protocol/unlock/assets/74358/32787acf-129a-4b48-a7fb-83db78992889)

3rd best option is to just use "Available" (that will already be better than current situation)

Reply:
We can't have 2 sperate repo and maintain 2 checkout modals :D We're barely able to do it with 1!

Reply:
We can do `10 Left` :) That's easy!